content/en/docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md

```diff
@@ -95,9 +95,9 @@ This limitation is tracked in [issue #55087](https://github.com/kubernetes/kuber
 
 ### Reference
 
-* [Lifecycle](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#lifecycle-v1-core)
-* [Container](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#container-v1-core)
-* See `terminationGracePeriodSeconds` in [PodSpec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core)
+* {{< api-reference page="workloads-resources/container" anchor="Lifecycle" >}}
+* {{< api-reference page="workloads-resources/container" >}}
+* See `terminationGracePeriodSeconds` in {{< api-reference page="workloads-resources/pod-v1" anchor="lifecycle" text="PodSpec" >}}
```

Comment:
Here is a view of the generated link (line 100):
https://deploy-preview-23294--kubernetes-io-master-staging.netlify.app/docs/tasks/configure-pod-container/attach-handler-lifecycle-event/#reference

So, `text` is actually the text displayed as the link.